### ISSUE TITLE
ClassCastException with mekanism transporter

### ISSUE THREAD

**Issue by bigenergy** [refinedmods/refinedstorage#3376]
### Describe the bug

Crash when loading chunks

### How can we reproduce this bug or crash?

![image](https://user-images.githubusercontent.com/17977139/191126597-9e986916-31fe-482f-b9ce-afb764d9660d.png)
where there is a stone, there was a pipe mekanizm

### What Minecraft version is this happening on?

Minecraft 1.18.2

### What Forge version is this happening on?

40.1.73

### What Refined Storage version is this happening on?

1.10.3

### Relevant log output

```shell
java.lang.ClassCastException: class com.refinedmods.refinedstorage.apiimpl.network.node.CableNetworkNode cannot be cast to class com.refinedmods.refinedstorage.apiimpl.network.node.InterfaceNetworkNode (com.refinedmods.refinedstorage.apiimpl.network.node.CableNetworkNode and com.refinedmods.refinedstorage.apiimpl.network.node.InterfaceNetworkNode are in module refinedstorage@1.10.3 of loader 'TRANSFORMER' @66756662)
	at com.refinedmods.refinedstorage.blockentity.InterfaceBlockEntity.lambda$new$0(InterfaceBlockEntity.java:22) ~[refinedstorage-1.10.3.jar%23132!/:1.10.3] {re:classloading}
	at net.minecraftforge.common.util.LazyOptional.getValue(LazyOptional.java:107) ~[forge-1.18.2-40.1.73-universal.jar%23167!/:?] {re:classloading,re:mixin}
	at net.minecraftforge.common.util.LazyOptional.getValueUnsafe(LazyOptional.java:119) ~[forge-1.18.2-40.1.73-universal.jar%23167!/:?] {re:classloading,re:mixin}
	at net.minecraftforge.common.util.LazyOptional.resolve(LazyOptional.java:227) ~[forge-1.18.2-40.1.73-universal.jar%23167!/:?] {re:classloading,re:mixin}
	at mekanism.common.content.network.InventoryNetwork.calculateAcceptors(InventoryNetwork.java:64) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.transporter.TransporterPathfinder.getPaths(TransporterPathfinder.java:49) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.transporter.TransporterPathfinder.getNewBasePath(TransporterPathfinder.java:103) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.transporter.TransporterStack.recalculatePath(TransporterStack.java:168) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.network.transmitter.LogisticalTransporterBase.lambda$insert$0(LogisticalTransporterBase.java:376) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.network.transmitter.LogisticalTransporterBase.insert(LogisticalTransporterBase.java:390) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.network.transmitter.LogisticalTransporterBase.insert(LogisticalTransporterBase.java:376) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.content.network.transmitter.LogisticalTransporterBase.onUpdateServer(LogisticalTransporterBase.java:123) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.tile.transmitter.TileEntityLogisticalTransporterBase.onUpdateServer(TileEntityLogisticalTransporterBase.java:32) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at mekanism.common.tile.transmitter.TileEntityTransmitter.tickServer(TileEntityTransmitter.java:89) ~[Mekanism-1.18.2-10.2.5.465.jar%23117!/:10.2.5] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:673) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:766) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:classloading}
	at net.minecraft.world.level.Level.redirect$zhn000$redirectTick(Level.java:1600) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:LevelMixin,pl:mixin:APP:observable.common.json:LevelMixin,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.WorldMixin,pl:mixin:APP:radium.mixins.json:world.block_entity_retrieval.WorldMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.WorldMixin,pl:mixin:APP:radium.mixins.json:world.inline_block_access.WorldMixin,pl:mixin:APP:radium.mixins.json:world.inline_height.WorldMixin,pl:mixin:A}
	at net.minecraft.world.level.Level.m_46463_(Level.java:476) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:LevelMixin,pl:mixin:APP:observable.common.json:LevelMixin,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.WorldMixin,pl:mixin:APP:radium.mixins.json:world.block_entity_retrieval.WorldMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.WorldMixin,pl:mixin:APP:radium.mixins.json:world.inline_block_access.WorldMixin,pl:mixin:APP:radium.mixins.json:world.inline_height.WorldMixin,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:327) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:gamepointguard.mixins.json:ServerWorldMixin,pl:mixin:APP:wyml.mixins.json:AccessorServerLevel,pl:mixin:APP:wyml.mixins.json:MixinServerWorld,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:APP:botania_xplat.mixins.json:MixinServerLevel,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:gamepointguard.mixins.json:MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:wyml.mixins.json:AccessorMinecraftServer,pl:mixin:APP:wyml.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:326) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:gamepointguard.mixins.json:MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:wyml.mixins.json:AccessorMinecraftServer,pl:mixin:APP:wyml.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:gamepointguard.mixins.json:MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:wyml.mixins.json:AccessorMinecraftServer,pl:mixin:APP:wyml.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23162!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:gamepointguard.mixins.json:MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:wyml.mixins.json:AccessorMinecraftServer,pl:mixin:APP:wyml.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}
```
https://gist.github.com/bigenergy/3fd990ae96b59680d57b012d448e9c2c

**Comment by raoulvdberge** [comment]
Duplicate of #3357

**Comment by raoulvdberge** [comment]
Hello. This bug has been fixed in v1.12.4 of Refined Storage. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
This bug has also been fixed in **v1.11.7** of Refined Storage for **Minecraft 1.19.2**. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
Hello, this bug is now also fixed for Minecraft 1.18.2 in v1.10.6.
